### PR TITLE
[Merged by Bors] - Fix voluntary exit to work with latest beacon api

### LINKED ISSUE #1932: Better information output on voluntary exit
## Description

The voluntary exit process is short on information that could be very useful to the user.

## Version

Lighthouse v0.3.4-46a0606
BLS Library: blst

## Present Behaviour

When doing a voluntary exit with the `account validator exit` command, no time is displayed. More information would also be welcomed.

## Expected Behaviour

It would be nice to display the current time of when the voluntary exit was published for logs and archiving. It would be nice to display when the expected exit is planned to actually happen (epoch and time delta). It would be nice to display some information saying that you still have to run your validator until the exit actually happens if you don't want to be penalized.

## Steps to resolve

- Change the output of the `account validator exit` command to include those information.
- An update to [the lighthouse-book page](https://lighthouse-book.sigmaprime.io/voluntary-exit.html) on this updated process would also be nice.

### PR DESCRIPTION
**Pull request by pawanjay176**
## Issue Addressed

Also fixes #1932 

## Proposed Changes

Use `ValidatorStatus::ActiveOngoing` instead of `ValidatorStatus::Active` to filter active validators.
Prints extra information regarding successful voluntary exit.

### CODE REVIEW

File: account_manager/src/validator/exit.rs [hunk 1]
```diff
@@ -165,6 +166,43 @@ async fn publish_voluntary_exit<E: EthSpec>(
             "Did not publish voluntary exit for validator {}. Please check that you entered the correct exit phrase.",
             keypair.pk
         );
+        return Ok(());
+    }
+
+    loop {
+        // Sleep for 1 slot duration and then check if voluntary exit was processed
+        // by checking the validator status.
+        // TODO: should we wait for longer to account for forks?
```
Comment (paulhauner) [inline account_manager/src/validator/exit.rs]: TODO

Perhaps we just update the "Voluntary exit has been accepted..." text to say something along the lines of:

> "Voluntary exit has been accepted into the beacon chain, but not yet finalized. Finalization may take several minutes or longer. Before finalization there is a low probability that the exit may be reverted."

**Review by paulhauner — CHANGES_REQUESTED** [review]
Nice, thanks for following up on this. There's a TODO in here, which I've commented. Perhaps we could also add some sort of message telling the user that we're going to wait until the exit is accepted into the chain and it might take several minutes. Perhaps adding a message when we poll but don't `break` would be handy too?

File: account_manager/src/validator/exit.rs (hunk 1 above)
Comment (pawanjay176) [reply account_manager/src/validator/exit.rs]: Yep, this sounds perfect. Will make the changes.

**Review by paulhauner — APPROVED** [review]
Nice one!

bors r+

**Comment by paulhauner** [comment]
I'll batch this

bors r-

**Pull request closed without merging**